containers.go

```diff
@@ -324,6 +324,9 @@ func (r *containerStore) Create(id string, names []string, image, layer, metadat
 				fmt.Sprintf("the container name \"%s\" is already in use by \"%s\". You have to remove that container to be able to reuse that name.", name, r.byname[name].ID))
 		}
 	}
+	if hasOverlappingRanges(options.UIDMap) || hasOverlappingRanges(options.GIDMap) {
+		return nil, errors.Wrapf(ErrInvalidMappings, "the specified mappings are overlapping")
```

Comment:
Could you change this error to

the specified UID and/or GID mappings are overlapping.